script/zonemaster-cli

```diff
@@ -255,8 +255,8 @@ level that were logged during the run.
 
 =item B<--[no-]nstimes>
 
-Print a summary, at the end of a run, of the times (in milliseconds) the zone's
-name servers took to answer.
+Print a summary, at the end of a run, of the times (in milliseconds) the name servers
+took to answer, as well as the number of sent queries.
```

Comment:
As I understand it, the time each name server takes it the total time over all queries.

```suggestion
Print a summary, at the end of a run, of the total time (in milliseconds) each name server
took to answer all queries, as well as the number of sent queries per name server.
```